libraries/helpers.rb

```diff
@@ -87,7 +87,7 @@ def ensure_mixlib_install_gem_installed!
             'mixlib-install'
           )
         else
-          install_gem_from_rubygems('mixlib-install', '1.0.6')
+          install_gem_from_rubygems('mixlib-install', '1.0.7')
```

Comment:
That's a good point.  I think I'm just following an annoying pattern right now from the "alpha" days.